### DISCUSSION TITLE
quill-better-table support for vue3 now ?

### DISCUSSION THREAD

**Discussion by tourlek** [vueup/vue-quill#114] [Q&A]
I want to use module quill-better-table on my project. but error

TypeError: Cannot read properties of undefined (reading 'pop')

quill.keyboard.bindings['Backspace'].pop()

```
"@vueup/vue-quill": "^1.0.0-beta.7",
"quill-2.0-dev": "^2.0.0-dev.3",
 "quill-better-table": "^1.2.10"
```

**Comment by loves990** [comment]
me too....
